.env.example

```diff
@@ -8,7 +8,7 @@
 NEXT_PUBLIC_LICENSE_CONSENT=''
 
 # DATABASE_URL='postgresql://<user>:<pass>@<db-host>:<db-port>/<db-name>'
-DATABASE_URL="postgresql://postgres:@localhost:5432/calendso?schema=public"
+DATABASE_URL="postgresql://postgres:@localhost:5450/calendso"
```

Comment:
why this?

Reply:
Just added comments @PeerRich

Reply:
Basically this is going to be used in `yarn dx`

Reply:
To prevent potential conflicts with local postgresql installs? I don't see why we should differ from the default psql port.